exercises/bracket-push/example.scala

```diff
@@ -0,0 +1,23 @@
+import scala.util.parsing.combinator.RegexParsers
+
+object Brackets extends RegexParsers {
```

Comment:
I just went searching around the site documentation and found the following -

> Reference Solution

> The reference solution is named something with example or Example in the path.

> The solution does not need to be particularly great code, it is only used to verify that the  exercise is coherent.

So, it sounds like having an overly complex example is fine:)

Thanks again for your feedback!

> If you change the test suite, then make sure the reference solution is fixed to pass the  updated tests.